code/modules/mob/new_player/new_player.dm

```diff
@@ -356,7 +356,7 @@
 	if(SSticker.random_players)
 		new_character.gender = pick(MALE, FEMALE)
 		client.prefs.real_first_name = random_first_name(new_character.gender)
-		client.prefs.real_last_name = random_first_name(new_character.gender)
+		client.prefs.real_last_name = random_last_name(new_character.gender)
```

Comment:
This is an unrelated fix to an oversight that gives people two random first names instead of a random first name and last name.